apps/studio/components/grid/components/grid/Grid.tsx

```diff
@@ -97,7 +99,28 @@ export const Grid = memo(
         })
       }
 
+      const [selectedCell, setSelectedCell] = useState<{ rowIdx: number; row: any; column: any }>()
```

Comment:
I'm wondering if we could get away with using `useRef()` here instead of `useState()`? Does this state need to be part of the React lifecycle for anything? Swapping it would reduce unnecessary rerenders.

Reply:
good catch, prob better to avoid useState in this one. thanks!